datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/manifest.py

```diff
@@ -0,0 +1,203 @@
+    "metrics": "Category::Metrics",
+    "monitoring": "Category::Monitoring",
+    "network": "Category::Network",
+    "notification": "Category::Notification",
+    "oracle": "Category::Oracle",
+    "orchestration": "Category::Orchestration",
+    "os & system": "Category::OS System",
+    "processing": "Category::Processing",
+    "profiling": "Category::Profiling",
+    "provisioning": "Category::Provisioning",
+    "security": "Category::Security",
+    "snmp": "Category::SNMP",
+    "source control": "Category::Source Control",
+    "testing": "Category::Testing",
+    "web": "Category::Web",
+}
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='Manifest utilities')
+def manifest():
+    pass
+
+
+@manifest.command(context_settings=CONTEXT_SETTINGS, short_help='Migrate a manifest to a newer schema version')
+@click.argument('integration', required=True)
+@click.argument('to_version', required=True)
+@click.pass_context
+def migrate(ctx, integration, to_version):
+    """
+    Helper tool to ease the migration of a manifest to a newer version, auto-filling fields when possible
+
+    Inputs:
+
+    integration: The name of the integration folder to perform the migration on
+
+    to_version: The schema version to upgrade the manifest to
+    """
+    echo_info(f"Migrating {integration} manifest to {to_version}....", nl=True)
+
+    # Perform input validations
+    if integration and integration not in get_valid_integrations():
+        abort(f'    Unknown integration `{integration}`, is your repo set properly in `ddev config`?')
+
+    loaded_manifest = JSONDict(load_manifest(integration))
+    manifest_version = loaded_manifest.get_path("/manifest_version")
+
+    if to_version not in SUPPORTED_MANIFEST_VERSIONS:
+        abort(f"    Unknown to_version: `{to_version}`. Valid options are: {SUPPORTED_MANIFEST_VERSIONS}")
+    if to_version == manifest_version:
+        abort(f"    {integration} is already on version `{manifest_version}`")
+    if to_version not in SUPPORTED_VERSION_UPGRADE_PATHS.get(manifest_version, []):
+        abort(
+            f"    Can't migrate from version `{manifest_version}` to version: `{to_version}`. Unsupported upgrade path"
+        )
+
+    migrated_manifest = JSONDict()
+    for key, val in V2_TO_V1_MAP.items():
+        if val == SKIP_IF_FOUND:
+            continue
+        # If the value is a string and is a JSONPath, then load the value from the JSON Path
+        elif isinstance(val, str) and val.startswith("/"):
+            final_value = loaded_manifest.get_path(val)
+        else:
+            final_value = val
+
+        if final_value is not None:
+            migrated_manifest.set_path(key, final_value)
+
+    # Update any previously skipped field in which we can use logic to assume the value of
+    # Also iterate through any lists to include new/updated fields at each index of the list
+    migrated_manifest.set_path("/manifest_version", "2.0.0")
```

Comment:
Last nit - can this line be moved just below line 152?

Since dicts remember insertion order, this will ensure that the version identifier always ends up at the top which will be helpful for quickly seeing the version.